verilog/formatting/tree_unwrapper.cc

```diff
@@ -469,7 +469,9 @@ void TreeUnwrapper::LookAheadBeyondCurrentLeaf() {
   while (!NextUnfilteredToken()->isEOF()) {
     EatSpaces();
     VLOG(4) << "lookahead token: " << VerboseToken(*NextUnfilteredToken());
-    if (IsComment(verilog_tokentype(NextUnfilteredToken()->token_enum()))) {
+    if (IsComment(verilog_tokentype(NextUnfilteredToken()->token_enum())) ||
```

Comment:
To avoid having the complicated expression to get the enum evaluate twice (and for the reader: read twice), we could to it possibly this way:

```c++
 if (auto next_tok = verilog_tokentype(NextUnfilteredToken()->token_enum());
     IsComment(next_tok) || next_tok == verilog_tokentype::TK_LINE_CONT) {  ...
```

Reply:
Thanks for the tip, it looks way better now